src/StrawberryShake/Client/src/Transport.WebSockets/ISocketConnectionPool.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace StrawberryShake.Transport
+{
+    public interface ISocketClientPool
```

Comment:
We do not longer pool connections but rather directly the socketclient as there is only one client per url